platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -657,48 +686,49 @@ check_free_memory() {
     cmd_res="$(need_cmd free)"
     if [ $? -ne 0 ]; then
         wrap_warn "check_free_memory"
-        wrap_warning "Could not find free utility to calculate current free memory. Skipping the check"
+        wrap_warning_message "Could not find free utility to calculate current free memory. Skipping the check"
         return
     fi
 
     eval iotedge_binary_memory='$'"$(echo "$ARCH"_iotedge_binaries_avg_memory)"
     eval iotedge_container_memory='$'"$(echo "$ARCH"_iotedge_container_memory)"
     total_iotedge_memory_size=$(echo $iotedge_binary_memory $iotedge_container_memory $iotedge_memory_buffer | awk '{print $1 + $2 + $3}')
     current_free_memory=$(free -m | awk '/^Mem/ {print $4}')
-    
+
     #TODO: correct final link of aka.ms/iotedge with the setup info of memory analysis.
     base_message="IoT Edge requires a minimum memory of $total_iotedge_memory_size MB for running the default setup as described in aka.ms/iotedge. We verified that the the device has $current_free_memory MB of free memory"
 
     res=$(echo $current_free_memory $total_iotedge_memory_size | awk '{if ($1 > $2) print 1; else print 0}')
     if [ $res -eq 1 ]; then
         wrap_pass "check_free_memory"
-        wrap_warning "$base_message"
+        wrap_warning_message "$base_message"
     else
         # TODO: Need to refine this message
         wrap_fail "Current available memory is $current_free_memory. Free up atleast $total_iotedge_memory_size MB to run IoT edge"
```

Comment:
What's the unit of the $current_free_memory?

Reply:
MB, will add

Reply:
Done